e2e/nomostest/nt.go

```diff
@@ -749,30 +757,63 @@ func (nt *NT) portForwardGitServer() {
 
 // portForwardRegistryServer forwards the registry-server deployment to a port.
 func (nt *NT) portForwardRegistryServer() {
-	provider := nt.OCIProvider.(*registryproviders.LocalOCIProvider)
-	resetImages := func(newPort int, podName string) {
-		for _, image := range nt.ociImages {
-			// construct push URL with provided port. Calling LocalPort would lead to deadlock
-			if err := image.Push(provider.PushURLWithPort(newPort, image.Name)); err != nil {
-				nt.T.Fatalf("pushing image: %w", err)
+	var onReadyCallbacks []func(int, string)
```

Comment:
need a comment about what this does. maybe "The local registry uses ephemeral storage. So re-push all the oci images and helm charts after the registry comes up."

Reply:
done